dashboard/app/entities_datastore.go

```diff
@@ -59,6 +59,7 @@ type Asset struct {
 	Type        dashapi.AssetType
 	DownloadURL string
 	CreateDate  time.Time
+	FsckLog     int64 // references to fsck logstext entity
```

Comment:
We should be able to distinguish between the cases when:
1) fsck was not run at all
2) fsck was run and it said the image is okay
3) fsck was run and found problems in the image

So we might need a bit more than just `FsckLog`.

Reply:
It was already somewhat possible with FsckLog being 0 meaning that fsck was not run and FsckLog containing "exited with code 0" meaning that the image is okay. I added a FsIsClean bool to avoid having to parse the log and I clarified that a FsckLog=0 means that fsck was not run.